src/components/Greetings/Greetings.jsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react'
+
+function Greetings(props) {
+  let {lang, children} = props
+
+  switch(lang){
+    case "de":
+      lang = "Hallo";
+      break;
+    case "fr":
+      lang = "Bonjour";
+      break;
+    case "en":
+      lang = "Hello";
+      break;
+    case "es":
+      lang = "Hola";
+      break;
+      default:
+        return "DKFDDHVAM";
+        
+  }
+   return (
+    <p>{lang} {children}</p>
+   )
+}
```

Comment:
Here, you are destructuring props. You shouldn't reassign a value to a prop. Instead, you should either create a separate variable or returning normal HTML